ratis-server/src/main/java/org/apache/ratis/server/impl/LeaderElection.java

```diff
@@ -271,6 +272,9 @@ private boolean shouldRun(long electionTerm) {
 
   private ResultAndTerm submitRequestAndWaitResult(Phase phase, RaftConfigurationImpl conf, long electionTerm)
       throws InterruptedException {
+    if (!conf.containsInConf(server.getId()) && phase == Phase.ELECTION) {
```

Comment:
We should fix the unit tests but not adding the phase check here.

Reply:
> We should fix the unit tests but not adding the phase check here.

OK, let's find out what's wrong with UT

Reply:
Could you help approve the workflow? : )

Reply:
@Xushaohong , the build finished.  Please take a look.